CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+
```

Comment:
Did you add this manually, or was this added by running `antsibull-changelog generate`?

Reply:
`CHANGELOG.rst` was only modified by  `antsibull-changelog generate`